lib/cylc/dump.py

```diff
@@ -32,8 +31,6 @@ def get_stop_state_summary(lines):
     for line in list(lines):
         if line.startswith('Remote command'):
             lines.remove(line)
-    line0 = lines.pop(0)
```

Comment:
Although `line0` is not used, `lines` is still used below, so we'll still need to remove the first element from it.